components/Button.tsx

```diff
@@ -3,7 +3,7 @@ import styled from "styled-components";
 //Typescript
 type ButtonProps = {
   name?: string;
-  onClick(event: React.MouseEvent<HTMLButtonElement>): void;
+  onClick?(event: React.MouseEvent<HTMLButtonElement>): void;
   className?: string;
```

Comment:
I can't comment on lines that are not part of the PR, so this is regarding the `Button` component in general:

I think it is not necessary to wrap your styled component (`StyledButton`)  inside another component (`Button`). You could export your styled component directly and it would do exactly the same.

Reply:
I don't think I understand you correctly :(

Reply:
Your `Button` component does nothing extra - it just displays a `StyledButton`. So you could simply remove your `Button` component,  rename the `StyledButton` to `Button` and export it.